src/index.tsx

```diff
@@ -17,13 +18,25 @@ export interface ZageProps {
   setShowZage: (b: boolean) => void;
   className?: string;
 }
-
+const oAuthUrls = ['bankofamerica', 'chase', 'wellsfargo', 'usbank', 'capitalone', 'usaa'];
```

Comment:
These are all plaid apps with oAuth integration, right? Two things
1. If Plaid adds an oAuth integration, will we need to update this list?
2. Does plaid do this remotely or will we need to update plaid so that way we never run into an issue where plaid has an oauth integration but we don't have their url marked?

Reply:
I added a default case that checks the url for a certain parameter which should make the easy 

Plaid does this remotely so not an issue for us to change anything